source/features/add-time-to-hovercard.js

```diff
@@ -60,5 +63,5 @@ async function updateHovercard() {
 }
 
 export default async function () {
-	delegate('[data-hydro-view]', 'view', updateHovercard);
+	observeEl('.Popover-message', updateHovercard);
```

Comment:
@bfred-it Curious, why did you make this switch back?

Reply:
They dropped the `view` event